client/src/App.js

```diff
@@ -36,6 +39,10 @@ function App() {
         });
       } else {
         setUserLoading(false);
+        socket.emit('userActive', { userId: user._id });
+        socket.on('makeActive', async () => {
+          fetch('/users/active').then((response) => {});
+        });
```

Comment:
```suggestion
        socket.on('makeActive', () => {
          fetch('/users/active').then((response) => {});
        });
```

You don't need the async if you're going to use a [.then](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises)

Reply:
But there's already some code in there that is using await
await friends.getFollowers(user._id, (err) => {
          throw new Error(err.message);
        });

I did use fetch() without then, but it raised an error.

Reply:
I think the `friends.getFollowers(...)` is considered un-related since it is in the outer scope of `socket.on('makeActive', async () => {...}`. So promise handling doesn't have to be the same.

If you want to use `async/await`, I think you could do [what this guy did](https://stackoverflow.com/a/47035468/7974948), and turn it to this:
```JavaScript
socket.on('makeActive', async () => {
  await fetch('/users/active');
});
```

I tried it and it worked on my local